client/my-sites/plan-features/actions.jsx

```diff
@@ -47,7 +47,7 @@ const PlanFeaturesActions = ( {
 		{
 			'is-current': current,
 			'is-primary': selectedPlan
-				? planType === selectedPlan
+				? planType === selectedPlan || planType === selectedPlan.concat( '_monthly' )
```

Comment:
I think we shouldn't be doing this. Monthly and yearly plans are completely separate plans, so we should highlight only the plan that we actually want to advertise (the one coming from the parameter).

Reply:
I tend to agree with the spirit of this change. I think that we generally want to highlight a plan independent of it's interval in the spirit that we're selling a plan as a sort of bundle of features, and they're the same for yearly and monthly versions of the same plan.

Is there any mechanism to highlight multiple plans? Is it even possible to say "highlight the monthly and yearly Jetpack Personal plans"?

It might be best to get some Growth opinions on this part of the change.

---

The logic here does concern me. We should be able to pass any plan (which may already be a monthly version) and have it match correctly. I don't believe that would match a yearly version as I believe is intended. It also depends on a plan slug convention which may or may not be maintained in the future.

I do believe there are some utilities already around this type of "plan matching". `client/lib/plans` is a good place to start looking.

Reply:
Thank you for the review @tyxla 👍 

Upgrade nudges advertise features and plan name ( Pro and Premium), not the way we charge for them, e.g. users come to plans with an intention to upgrade to Pro because of SEO Tools. Why would we not show that in the `monthly` view but only the `yearly` (which happens only because one of them needs to be the default one :) )? 
Am I missing on some important consequences here ?

Reply:
I take my comment back. Seems like we're currently advertising the Personal plan for both monthly and yearly, so I'm fine with this change. Let's make sure to ask @richardmuscat just in case.

I support what @sirreal mentioned here that we should rely on a more common method to compare plans - we've been encountering needs for such utility numerous times already.

Reply:
Sounds like a great idea- I'll try to spec out something most reusable for this.

Reply:
I believe this is all we need: https://github.com/Automattic/wp-calypso/blob/72bce7158b76ebd7a7546ee9faf60624a47a122b/client/lib/plans/index.js#L204-L222

Reply:
Thanks @sirreal!